docs/install/README.md

```diff
@@ -64,7 +64,7 @@ quickly.
 
 ## Installing Knative with Kourier
 
-[Installing Knative with Kourier](./knative-with-kourier.md): In this guide, Knative will be installed with Kourier to replace the Istio Service Mesh, providing a more lightweight and simple Ingress controller. Note that Istio is required for the Knative Eventing component.
+[Installing Knative with Kourier](./knative-with-kourier.md): In this guide, Knative will be installed with Kourier to replace the Istio Service Mesh, providing a more lightweight and simple Ingress controller. Note that Istio can serve as an Ingress controller for the Knative Serving component as well.
```

Comment:
If it isn't required and originally mentioned Eventing, maybe we can just delete the note instead of adding additional info about serving. Above this section is "Installing Knative with Istio" which should cover the content of the note.

Reply:
+1 to removing text if it's no longer needed. If Kourier can fulfil the cluster-local routes, I don't think we need any note about Istio here.